src/main/java/mate/academy/internetshop/dao/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,151 @@
+package mate.academy.internetshop.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import mate.academy.internetshop.dao.OrderDao;
+import mate.academy.internetshop.lib.Dao;
+import mate.academy.internetshop.model.Item;
+import mate.academy.internetshop.model.Order;
+import org.apache.log4j.Logger;
+
+@Dao
+public class OrderDaoJdbcImpl extends AbstractDao<Order> implements OrderDao {
+    private static Logger logger = Logger.getLogger(OrderDaoJdbcImpl.class);
+
+    public OrderDaoJdbcImpl(Connection connection) {
+        super(connection);
+    }
+
+    public List<Item> addItemsInDb(Order order) {
```

Comment:
```suggestion
    private List<Item> addItemsInDb(Order order) {
```
@TatianaPirch can this method be private?